apps/webapp/src/state/send.ts

```diff
@@ -133,13 +133,11 @@ const planWitnessBuildBroadcast = async ({ amount, recipient, asset }: SendSlice
   return uint8ArrayToHex(id.hash);
 };
 
-export const amountToBig = (asset: TempAsset, assetBalance: Amount) => {
-  const exponent = asset.denomUnits.find(d => d.denom === asset.display)!.exponent;
-  return fromBaseUnitAmount(assetBalance, exponent);
-};
+export const amountToBig = (asset: AssetBalance) =>
```

Comment:
Same with this function. Can just move `fromBaseUnitAmount(...)` where it's used